gotham/src/helpers/http/response/mod.rs

```diff
@@ -64,10 +64,22 @@ type Body = (Vec<u8>, Mime);
 /// #     );
 /// # }
 /// ```
-pub fn create_response(state: &State, status: StatusCode, body: Option<Body>) -> Response {
-    let mut res = Response::new();
-    extend_response(state, &mut res, status, body);
-    res
+pub fn create_response(
+    state: &State,
+    status: StatusCode,
+    body: Option<DataMime>,
+) -> Response<Body> {
+    let mut builder = Response::builder();
+    let mime = body.clone().map(|(_, mime)| mime);
```

Comment:
I'm not sure if this can be done in such a way as to avoid the clone; maybe unpack it and clone the mime type itself? The vec in body could be potentially quite large.